yb-voyager/cmd/export.go

```diff
@@ -246,6 +268,31 @@ func validateSourceDBType() {
 	}
 }
 
+func validateConflictsBetweenTableListFlags(tableList string, excludeTableList string) {
```

Comment:
You will need to simplify this function when you support both `--table-list` and `--exclude-table-list` in the same command (as we discussed).